code/edgebridge/build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
+    id("aep-library")
+}
+
+val mavenCoreVersion: String by project
+val androidxAnnotationVersion: String by project
+
+aepLibrary {
+    namespace = "com.adobe.marketing.mobile.edge.bridge"
+
+    publishing {
+        gitRepoName = "aepsdk-edgebridge-android"
+        addCoreDependency(mavenCoreVersion)
+    }
+
+    enableSpotless = true
+    enableSpotlessPrettierForJava = true
+}
+
+dependencies {
+    // TODO: Use 3.x versions for testing
+    implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion")
```

Comment:
nit: 
Update `mavenCoreVersion` to 3.0.0 and append the [`-snapshot` suffix](https://github.com/adobe/aepsdk-userprofile-android/blob/dev-v3.0.0/code/userprofile/build.gradle.kts#L20)  to ensure that the CI builds it with most recent Core version. You can remove the suffix once we release Core.